CSS/style.css

```diff
@@ -42,12 +42,11 @@ header {
     width: 600px;
 }
 
-.icon-middle > button {
-    /* THIS NEEDS TO BE FIXED */
+/* .icon-middle > button {
     height: 26px;
     margin-left: 0;
     width: 65px;
-}
+} */
```

Comment:
Please delete code that is no longer useful instead of commenting it out. If you want to view previous versions of your code, use git history instead :+1:

Reply:
Deleted not useful code